src/box/memtx_tree.cc

```diff
@@ -1981,18 +2012,23 @@ static int
 tree_read_view_create_iterator(struct index_read_view *base,
 			       enum iterator_type type,
 			       const char *key, uint32_t part_count,
+			       const char *pos,
 			       struct index_read_view_iterator *iterator)
 {
 	struct tree_read_view_iterator<USE_HINT> *it =
 		(struct tree_read_view_iterator<USE_HINT> *)iterator;
 	it->base.index = base;
 	it->base.next_raw = exhausted_index_read_view_iterator_next_raw;
+	it->base.position = generic_index_read_view_iterator_position;
```

Comment:
Nit: This function should never be called in CE so it's better to use a callback that resolves to `unreachable()`.

Reply:
Outdated - we set actual position method here now.